samcli/lib/utils/git_repo.py

```diff
@@ -129,8 +129,11 @@ def clone(self, clone_dir: Path, clone_name: str, replace_existing: bool = False
                 temp_path = os.path.normpath(os.path.join(tempdir, clone_name))
                 git_executable: str = GitRepo._git_executable()
                 LOG.info("\nCloning from %s (process may take a moment)", self.url)
+                command = [git_executable, "clone", self.url, clone_name]
+                if platform.system().lower() == "windows":
+                    command += ["--config", "core.longpaths=true"]
```

Comment:
Users might need to enable long path settings in windows registry as well. Should we add a debug message to remind that too?

Reply:
yes, we should. Will update it.